Tiltfile

```diff
@@ -454,6 +478,62 @@ def waitforsystem():
     local(kubectl_cmd + " wait --for=condition=ready --timeout=300s pod --all -n capi-kubeadm-control-plane-system")
     local(kubectl_cmd + " wait --for=condition=ready --timeout=300s pod --all -n capi-system")
 
+def peer_vnets():
+    # TODO: check for az cli to be installed in local
+    # wait for AKS VNet to be in the state created
+    peering_cmd = "; echo \"--------Peering VNETs--------\""
```

Comment:
Could we maybe use multiline [string literals](https://github.com/bazelbuild/starlark/blob/master/spec.md#string-literals) to make this more readable?

```starlark
peering_cmd = """; echo "--------Peering VNETs--------"
; az network vnet wait --resource-group ${AKS_RESOURCE_GROUP} --name ${AKS_MGMT_VNET_NAME} --created --timeout 180
; export MGMT_VNET_ID=$(az network vnet show --resource-group ${AKS_RESOURCE_GROUP} --name ${AKS_MGMT_VNET_NAME} --query id --output tsv)
; echo " 1/8 ${AKS_MGMT_VNET_NAME} found "
"""
```

Reply:
Thank you for pointing multi-line string literal. I did not use it here because multi-line string literal made it harder for me to debug/print 😅.

Plus, this code block is a result of iterative development, hence each command is broken over multiple lines. 

I can put it in multi-line string literal if needed. What do you say ?

Reply:
I think it would be easier to read if each group (each section starting with a comment) were one multiline string. So you'd still be doing string concatenation, but with longer, logically coherent strings without escape characters. But it's mostly a style thing, lgtm otherwise.

Reply:
Updating it so :)